### PR TITLE
chore: adapt to naming convention of sdk examples

### LINKED ISSUE #402: rename `client` to `aeSdk` for variables and functions
we want to keep this consistent in all future examples.

see: https://github.com/aeternity/aepp-sdk-js/issues/1300#issuecomment-1014203920

### PR DESCRIPTION
**Pull request by marc0olo**
closes #402 

tests fail until https://github.com/aeternity/aeproject/issues/404 is resolved

### CODE REVIEW

File: docs/migration-from-3.x.x-to-4.x.x.md [hunk 1]
```diff
@@ -63,4 +63,4 @@ const { networks, utils, wallets } = require('@aeternity/aeproject');
         - local testing network is now `devmode` instead of `local`
     - replace `defaultWallets` import with `const { wallets } = require('@aeternity/aeproject');` for prefunded wallets
     - replace `contractUtils` import with `const { utils } = require('@aeternity/aeproject');` for utils
-        - consider using the new helpers for getting a client and creating snapshots similar to `test/exampleTest.js`
+        - consider using the new helpers for initializing the SDK and creating snapshots similar to `test/exampleTest.js`
```
Comment (thepiwo) [inline docs/migration-from-3.x.x-to-4.x.x.md:66]: technically incorrect "initializing an instance of the sdk"

File: src/init/update-artifacts/test/exampleTest.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ const { utils, wallets } = require('@aeternity/aeproject');
 const EXAMPLE_CONTRACT_SOURCE = './contracts/ExampleContract.aes';
 
 describe('ExampleContract', () => {
-  let client;
+  let aeSdk;
   let contract;
 
   before(async () => {
-    client = await utils.getClient();
+    aeSdk = await utils.getSdk();
```
Comment (thepiwo) [inline src/init/update-artifacts/test/exampleTest.js:11]: technically incorrect, its an initialized instance of the sdk we are getting, not the sdk itself

**Review by thepiwo — COMMENTED** [review]
well, I voiced my concerns regarding this renaming already, so feel free to merge without my approval.

File: src/init/update-artifacts/test/exampleTest.js (hunk 2 above)
Comment (marc0olo) [reply src/init/update-artifacts/test/exampleTest.js:11]: I'd really like to not discuss this until 2023. back then I got almost no response in https://github.com/aeternity/aepp-sdk-js/issues/1300

would you be fine with `aeSdk = await utils.getSdkInstance();` ?

Comment (marc0olo) [reply src/init/update-artifacts/test/exampleTest.js:11]: though in the docs we describe it already like this:

- `getSdk()` get an instance of the SDK for local development
        - initialized with all prefunded wallets for `onAccount` to be used calling from different accounts

Comment (marc0olo) [reply src/init/update-artifacts/test/exampleTest.js:11]: I really wanna avoid discussing `aeSdk`

Comment (thepiwo) [reply src/init/update-artifacts/test/exampleTest.js:11]: > I really wanna avoid discussing `aeSdk`

that was my feedback all along, but I guess its not my decision

Comment (marc0olo) [reply src/init/update-artifacts/test/exampleTest.js:11]: I heard it the first time some weeks ago and I didn't notice any other comment of your side back then when we discussed that. if you told this in some chat I just cannot remember.

now - at least for the moment - I'd be happy if we can go this route.

maybe `client` is technically more correct. but it's also not explicitly telling us what it's about (at least right now).

I totally get your point, but I don't see it that strict. I would propose that we use the governance aepp in the future if our dev community gets bigger to vote on this 😅 I'd actually like such vote. but currently it wouldn't make sense IMO.

**Comment by marc0olo** [comment]
> well, I voiced my concerns regarding this renaming already, so feel free to merge without my approval.

let's go that route for the moment as discussed and already adapted in SDK. if we aim to change this again at some time we should make sure everybody is fine with the route we choose.

**Pull request merged**